data/crac-io/crac-io-json/src/test/java/com/powsybl/openrao/data/craciojson/JsonSerializationConstantsTest.java

```diff
@@ -174,10 +174,10 @@ void testUsageRuleComparatorMix() {
         assertTrue(comparator.compare(ovc2, ovc1) < 0);
 
         assertTrue(comparator.compare(oi1, ocs1) > 0);
-        assertTrue(comparator.compare(ocs1, ofc1) < 0);
+        assertTrue(comparator.compare(ocs1, ofc1) > 0);
```

Comment:
Imo this method is used to sort the usage methods when the CRAC is serialized
The test is here to ensure that it works properly
I had to change this because there is no longer the specific onConstraint usage rules